network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/typeform_block.html

```diff
@@ -1,7 +1,14 @@
-<a
-  id="btn-typeform-popup"
-  class="btn {{ value.button_type }} my-3"
-  href="https://form.typeform.com/to/{{ value.embed_id }}"
-  target="_blank">{{ value.button_text }}</a>
 
-<script async defer src="https://embed.typeform.com/embed.js" ></script>
+<div class="container {{block.block_type}}-block">
+  <div class="row justify-content-center">
+    <div class="streamfield-content">
+      <a
+        id="btn-typeform-popup"
+        class="btn {{ value.button_type }} my-3"
+        href="https://form.typeform.com/to/{{ value.embed_id }}"
+        target="_blank">{{ value.button_text }}</a>
+
+      <script async defer src="https://embed.typeform.com/embed.js" ></script>
```

Comment:
Much like the airtable block, we may need a follow-up issue here because if a page happens to include two of these blocks, that's going to load the JS twice. Not a problem right now, but something we should have a cleanup issue for.